terraform/development/shared/main.tf

```diff
@@ -85,10 +85,11 @@ module "application_roles" {
 
   github_actions_role_arn = "arn:aws:iam::815624722760:role/core-application-repo"
 
-  prefix                               = local.prefix
-  bulk_upload_bucket_access_policy_arn = module.bulk_upload.read_write_policy_arn
-  database_data_access_policy_arn      = module.database.rds_data_access_policy_arn
-  export_bucket_access_policy_arn      = module.cds_export.read_write_policy_arn
+  prefix                                        = local.prefix
+  bulk_upload_bucket_access_policy_arn          = module.bulk_upload.read_write_policy_arn
+  collection_resources_bucket_access_policy_arn = module.collection_resources.details.bucket_name

```

Comment:
is `module.collection_resources.details.bucket_name` correct?

Reply:
No I'll change that